source/guide_teamspeak.rst

```diff
@@ -0,0 +1,286 @@
+  2020-02-14 21:04:59.245200|INFO    |CIDRManager   |   |updated query_ip_whitelist ips: 127.0.0.1/32, ::1/128, 
+
+
+You can now press **CTRL+C** to kill the server and to continue with the configuration. 
+
+.. note:: Make a copy of **Server Query Admin Account** ``loginname``, ``password`` and **ServerAdmin** ``token``. You vill need this when you connect to your teamspeak server for the first time to gain administrator rights.
+
+Step 3: Edit configuration file
+-------------------------------
+
+Now edit the ``ts3server.ini`` and change the following entries. Use the four ports you were assigned by ``uberspace port add`` above.
+
+::
+  
+  machine_id=0
+
+::
+  
+  default_voice_port=40132   
+
+::
+
+  filetransfer_port=40133
+
+::
+
+  query_port=40134
+
+::
+
+  license_accepted=1
+
+::
+
+  query_ssh_port=40135
+
+
+Step 4: Test server start
+-------------------------
+
+You can test your configuration by executing the minimal runscript. Use the paramater ``inifile`` with the argument ´´ts3server.ini`` to start the server with your configuration. 
+
+.. code-block:: console
+
+  [isabell@stardust ts3server]$ ./ts3server_minimal_runscript.sh inifile=ts3server.ini
+  2020-02-14 21:43:43.659472|INFO    |ServerLibPriv |   |TeamSpeak 3 Server 3.11.0 (2020-01-13 08:12:37)
+  [...]
+  2020-02-14 22:17:31.884829|INFO    |              |   |Puzzle precompute time: 703
+  2020-02-14 22:17:31.885977|INFO    |FileManager   |   |listening on 0.0.0.0:47304, [::]:47304
+  2020-02-14 22:17:31.895316|INFO    |VirtualServerBase|1  |listening on 
+  2020-02-14 22:17:31.896826|INFO    |Query         |   |listening for query on 0.0.0.0:47305, [::]:47305
+  2020-02-14 22:17:31.897214|INFO    |Query         |   |listening for query ssh on 0.0.0.0:47306, [::]:47306
+  2020-02-14 22:17:31.897479|INFO    |CIDRManager   |   |updated query_ip_whitelist ips: 127.0.0.1/32, ::1/128,
+
+.. note:: If this does not work right now, it might be, that the ports are not yet open. Try again in a couple minutes. 
+
+If you want, you can also try to connect to the teamspeak server with your local teamspeak client software. Please use the IP address of your server the the default_voice_port (<IPofYourServer>:<default_voice_port>)
+
+Example:
+
+  .. image:: _static/images/teamspeak_client_connect.png
+      :align: center
+
+
+Configure the supervisord service
+=================================
+
+Since teamspeak does come with a start-stop script, all we need to do is to create the ``supervisor`` ini-file.
+
+Create supervisord ini (e.g. ``/home/$USER/etc/services.d/ts3server-daemon.ini``):
+
+::
+  
+  [program:ts3server]
+  directory=%(ENV_HOME)s/ts3server/
+  command=%(ENV_HOME)s/ts3server/ts3server inifile=ts3server.ini
+  startsecs=10
+  autostart=yes
+  autorestart=yes
+
+After creating the configuration, tell supervisord to refresh its configuration and start the service:
+
+.. code-block:: console
```

Comment:
Please use `.. include:: includes/supervisord.rst` instead.

Reply:
updated and used the include.